src/locales/en.ts

```diff
@@ -119,6 +122,32 @@ export const en = {
     headerDescription3: 'only be shown once.',
     headerDescription4: 'Please copy this value and keep for your records.'
   },
+  analytics: {
+    filterLabelProductVersions: 'Product Versions',
+    chartOverview: 'Chart Overview',
+    chartTitleRequests: 'Requests by Product Version',
+    chartTitleLatency: 'P99 Latency by Product Version',
+    chartTitle4xxProductVersion: '4xx by Product Version',
+    chartTitle5xxProductVersion: '5xx by Product Version',
+    chartTitle4xxStatusCode: '4xx by Status Code',
+    chartTitle5xxStatusCode: '5xx by Status Code',
+    dashboard: 'Dashboard',
+    resultsLimited: 'Not all results shown. Refine your search for more related results.',
+    notAvailable: 'Not available',
+    sectionCurrent: 'Current',
+    sectionLast: 'Last',
+    sectionPrevious: 'Previous',
+    selectDateRange: 'Please select a date range',
+    selectProductVersions: 'Select Product Versions',
+    summary: 'Summary',
+    summary24Hours: '24 Hours',
+    summary30Days: '30 Days',
+    summaryTooltip: (timespan) => `Showing ${timespan} summary of analytics for all apps`,
```

Comment:
Huh: this package isn't using the `i18n` templating mechanism?

Reply:
Seems like this should work, given the existing instances 

https://github.com/Kong/konnect-portal/pull/133/files#diff-9655675aa170422d397e0c2c554bff9ab02cd7ec0411f0bfa45341d380bf1f01L108